### DISCUSSION TITLE
'Scan' operator runtime session error?

### DISCUSSION THREAD

**Discussion by shashwat1198** [microsoft/onnxruntime#16253] [Other Q&A] [answered]
Hello,

I have been trying to run an onnxruntime inference session for the 'Scan' operator in onnx using the addition example given in the documentation : https://github.com/onnx/onnx/blob/main/docs/Operators.md#Scan

When I try to run the session, I get the following error:
![ONNX error](https://github.com/microsoft/onnxruntime/assets/49102578/a5ed3584-b802-43b2-ba79-6129a3510bf2)

Can anyone point me as to how I can solve this issue? The model also passes the checker test and I don't see any error there.

Any help would be highly appreciated,
Shashwat

**Comment by xadupre** [comment]
How did you build your graph? These examples are of the onnx backend test and tested in onnx CI and onnxruntime CI (https://github.com/onnx/onnx/blob/main/onnx/test/test_backend_onnxruntime.py).

**Reply by shashwat1198** [reply, marked as answer]
Hi, Thanks for your response, I was eventually able to use the scan operator to implement an LSTM network available here: https://github.com/fastmachinelearning/qonnx/blob/main/notebooks/4_quant_lstm.ipynb
